### DISCUSSION TITLE
Variables to save in CLM6 PPE

### DISCUSSION THREAD

**Discussion by linniahawkins** [NCAR/ctsm6_ppe#3] [General]
We are currently saving standard output variables, if you have specific variables you would like us to add please comment below. 

- Standard gridcell variables monthly (1850-2023)
- PFT-level variables monthly (1850-2023)
- Daily gridcell level data (1985-2023)

[Google sheet with all variables](https://docs.google.com/spreadsheets/d/1IDBbKqKSSOaBY5J4RcS-TZKTYGndOV0oTN15WUco7xY/edit?usp=drive_link)

**Comment by katiedagon** [comment]
@minallah is interested in the following variables at daily timescales:

State: SOILLIQ + SOILICE + H2OSNO + H2OCAN + H2OSFC
Fluxes: RAIN + SNOW - QVEGT - QVEGE - QSOIL - QDRAI - QOVER - QRGWL - QDRAI_PERCH

**Comment by djk2120** [comment]
All those states look reasonable to me.

Re: the fluxes
rain we can get from the forcing, so it seems wasteful to write it for each
ensemble member,
does she need snow separately or is she interested in the total precip?
does she need all of the individual runoff components, or would their sum
(QRUNOFF) be sufficient?

I might suggest a smaller list of fluxes as:
QVEGT, QVEGE, QSOIL, QRUNOFF

Re: the forcing variables
It might be worth writing them all at the daily timestep, but only for the
default simulation

**Comment by linniahawkins** [comment]
We are currently saving these fluxes daily: QVEGT, QVEGE, QSOIL, QRUNOFF
and these states daily: State: SOILLIQ + SOILICE + H2OSNO + H2OSFC
so we just need to add H2OCAN
